### PR TITLE
Remove FIXME and link to docs for typed pub sub

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
thx

File: akka-docs/src/main/paradox/distributed-pub-sub.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Classic Distributed Publish Subscribe in Cluster
 
 @@include[includes.md](includes.md) { #actor-api }
-For the new API see FIXME https://github.com/akka/akka/issues/26338.
+
```
Comment (patriknw) [inline akka-docs/src/main/paradox/distributed-pub-sub.md]: in other places we don't have a blank line here

**Comment by raboof** [comment]
Failure is due to a problem on master, fixed by https://github.com/akka/akka/pull/28770

**Review by raboof — APPROVED** [review]

**Comment by johanandren** [comment]
PLS BUILD

**Pull request merged**